backend/internal/api/api_test.go

```diff
@@ -573,4 +577,34 @@ func TestDBPostHandler(t *testing.T) {
 		assert.Equal(t, http.StatusOK, recorder.Code)
 
 	})
+
+	t.Run("should log proper error when sending event fails", func(t *testing.T) {
+		//given
+		hook := test.NewGlobal()
+		value := `{` +
+			`"id":"` + fixedID + `",` +
+			`"content":"base64",` +
+			`"gcp":"",` +
+			`"status":false` +
+			`}`
+
+		req, err := http.NewRequest("POST", "/v1/images", bytes.NewBuffer([]byte(value)))
+		assert.NoError(t, err)
+		req.Header.Set("Content-Type", "application/json")
+		recorder := httptest.NewRecorder()
+		dbManagerMock := apiMocks.DBManager{}
+		idMock := utilsMocks.IdGenerator{}
+		idMock.On("NewID").Return(fixedID, nil)
+		testSubject := NewHandler(&dbManagerMock, &idMock)
+		dbManagerMock.On("InsertToDB", fixedID, value).Return(nil)
+		eventHelperMock := eventMocks.EventHelper{}
+		eventHelperMock.On("SendEvent").Return(errors.New("SENDEVENT: error"))
```

Comment:
This is just a general remark. Alternative approach to mocking is to create test http server that will respond with some fixed stuff. This is a matter of approach: you can mock or call some fake (fake http server in our case).